docs/user/whats-new.asciidoc

```diff
@@ -67,24 +66,48 @@ instead of changing the index for each individual element. For instructions, ref
 Compare two metrics on the y-axes at the same time.
 This enables you compare and contrast data of vastly different types and ranges.
 For example, you might track the total page views for your web site,
-which tallies in the millions. At the same time, you're also interested
+which tallies in the millions. But you're also interested
 in the percentage of the web site bounce rates,
 typically between 1 and 100. You can now stack both metrics on top of one
 another to quickly spot the outliers.
 
 Also in 7.9, Lens debuts color pick, sort, and a fitting function.
 
+[float]
+[[dashboard-7-9]]
+=== Dashboard to Discover explorations
+
+Navigating between dashboards was a highlight in 7.8, and 7.9 adds
+the ability to navigate from a dashboard to Discover.  The time range,
+filters, and other parameters remain the same, so you can continue your
+analysis from a new perspective. For example,
+your dashboard might have a visualization for overall HTTP responses.
+You can navigate to Discover to see the saved search
+for HTTP responses. The new {kibana-ref}/explore-underlying-data.html[*Explore underlying data*]
```

Comment:
This was disabled by default in [#74332](https://github.com/elastic/kibana/pull/74332).

Reply:
@hendry-lim there are two "Explore underlying data" actions.

1. One that appears in panel's context menu;
2. and one that appears in-chart on chart interaction.

The one in panel's context menu is *always enabled*, there is no way to disable it.

It is only possible to disable the second, in-chart, action. And it is disabled by default. I don't think we should overwhelm users with the specifics in this release note. I believe it is OK if they learn about the second action when they navigate to the docs of "Explore underlying action".